app/form_objects/support_form.rb

```diff
@@ -10,7 +12,8 @@ class SupportForm
   alias_attribute :question, :message
 
   validates :i_need_help_with, presence: true, inclusion: { in: I_NEED_HELP_WITH_OPTIONS }
-  validates :i_need_help_with, presence: true, inclusion: { in: I_NEED_HELP_WITH_OPTIONS - %w[other_government_service] }, on: :submit
+  validates :i_need_help_with, presence: true,
+                               inclusion: { in: I_NEED_HELP_WITH_OPTIONS - %w[other_government_service] }, on: :submit
```

Comment:
This feels harder to read to me. Can we tweal the wrapping by hand or is the idea that we always let the algorithm handle it?